### ISSUE TITLE
NTR: Lipoprotein X high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8260]
New term request Lipoprotein X high in blood (elthat_100517014512):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000566
- label: Elevated circulating lipoprotein X concentration
- parents: [Abnormal circulating apolipoprotein concentration (HP:0025201)](https://hpo.jax.org/app/browse/term/HP:0025201)
- pmid: PMID:30503707
- def: The concentration of lipoprotein X in the blood circulation is above the upper limit of normal.
- comment: The hypercholesterolemia observed in cholestasis is due to the presence of an anomalous lipoprotein called lipoprotein-X (Lp-X). Lp-X is a lipoprotein rich in phospholipid and free cholesterol present in plasma of patients with cholestasis and, with some variations, in patients with lecithin-cholesterol-acyl-transferase deficiency (LCAT), and after lipid infusion. Lp-X is formed from a bile lipoprotein moving to the blood vessels where it incorporates small quantities of triglycerides, apo-C and esterified cholesterol and becomes a mature Lp-X.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8260
- synonyms:
